src/Controls/src/Core/MergedStyle.cs

```diff
@@ -190,13 +198,17 @@ void SetStyle(IStyle implicitStyle, IList<Style> classStyles, IStyle style)
 			_classStyles = classStyles;
 			_style = style;
 
+			//FIXME compute specificity
```

Comment:
There are a few comments like this one, do they need to be fixed before we merge this?